docs/operator-manual/notifications/index.md

```diff
@@ -45,3 +45,67 @@ So you can just use them instead of reinventing new ones.
     ```
 
 Try syncing an application to get notified when the sync is completed.
+
+## Namespace based configuration
+
+!!! important
+Available since v2.9
+
+A common installation method for Argo CD Notifications is to install it in a dedicated namespace to manage a whole cluster. In this case, the administrator is the only
+person who can configure notifications in that namespace generally. However, in some cases, it is required to allow end-users to configure notifications
+for their Argo CD applications. For example, the end-user can configure notifications for their Argo CD application in the namespace where they have access to and their Argo CD application is running in.
+
+This feature is based on applications in any namespace. See [applications in any namespace](../app-any-namespace.md) page for more information.
+
+In order to enable this feature, the Argo CD administrator must reconfigure the argocd-notification-controller workloads to add  `--application-namespaces` and `--self-service-notification-enabled` parameters to the container's startup command.
+`--application-namespaces` controls the list of namespaces that Argo CD applications are in. `--self-service-notification-enabled` turns on this feature.
+
+The startup parameters for both can also be conveniently set up and kept in sync by specifying
+the `application.namespaces` and `notificationscontroller.selfservice.enabled` in the argocd-cmd-params-cm ConfigMap instead of changing the manifests for the respective workloads. For example:
+
+```yaml
+data:
```

Comment:
```suggestion
apiVersion: v1
kind: ConfigMap
metadata:
  name: argocd-cmd-params-cm
data:
```

Just to be a bit more explicit. I find it's easier to skim if these details are in the yaml.